site/content/help/early-access/index.md

```diff
@@ -70,7 +70,7 @@ You should see an option to select the `Early Access` release channel.
 >}}
 
 #### Step 7
-On the next page of the installer, paste the token that you previously copied in `Step 4`, and click `Verify Token.`
+On the next page of the installer, paste the token (highlight the `Token Entry Section` and press CTRL+V to paste) that you previously copied in `Step 4`, and click `Verify Token.`
```

Comment:
Would be better as: 

On the next page of the installer, paste the token you previously copied in `Step 4` by selecting the `Token Entry Box` and pressing CTRL + V on your keyboard, and then click `Verify Token` to authenticate.